### ISSUE TITLE
Bug: css variable not resolved in rule "css/no-invalid-properties"

### ISSUE THREAD

**Issue by Otlikov** [eslint/css#200]
### Environment

ESLint version: 9.31.0
@eslint/css version: 0.10.0
Node version: 24.4.0
npm version: 11.4.2
Operating System: macOS 15.5 (24F74)

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```
eslint css/no-invalid-properties: ["error"]
```
</details>

```css
/* eslint css/no-invalid-properties: ["error"] */

:root {
	--mySize: 80px;
	--myFontSize: calc(0.5 * var(--mySize));
}

.test {
	font-size: var(--myFontSize);
}
```

### What did you expect to happen?

No error as variable --myFontSize is properly defined.

### What actually happened?

Unknown property 'font-size' found  css/no-invalid-properties

### Link to Minimal Reproducible Example

none

### Participation

- [ ] I am willing to submit a pull request for this issue.
